itai_q3.py

```diff
@@ -1,11 +1,11 @@
-def three_times_number(lst):
-    dict = {}
-    for num in lst:
-        if num in dict:
-            dict[num] += 1
-            if dict[num] == 3:
-                return num
-        else:
-            dict[num] = 1
-    return "There isn't such a number in this list"
-print(three_times_number([]))
+def three_times_number(lst):
+    dict = {}
+    for num in lst:
+        if num in dict:
+            dict[num] += 1
+            if dict[num] == 3:
+                return num
+        else:
+            dict[num] = 1
```

Comment:
I would do it in a more elegant way:
```suggestion
        if num not in dict:
          dict[num] = 0
        dict[num] += 1
        if dict[num] == 3:
            return num
```